x-pack/plugins/apm/public/hooks/use_fetcher.tsx

```diff
@@ -179,14 +196,6 @@ export function useFetcher<TReturn>(
     /* eslint-enable react-hooks/exhaustive-deps */
   ]);
 
-  useEffect(() => {
-    if (result.error) {
-      addInspectorRequest({ ...result, data: result.error.body?.attributes });
-    } else {
-      addInspectorRequest(result);
-    }
-  }, [addInspectorRequest, result]);
-
```

Comment:
Usually these changes are better to do in separate PRs but it's relatively small so probably fine. Did you verify that this works?

Reply:
I agree, but yes, since it was small i pushed it with this change. 
Yes i verified the feature after this change